### DISCUSSION TITLE
Ability to choose renderer

### DISCUSSION THREAD

**Discussion by IronRevelation** [mrshmllow/affinity-nix#15] [Q&A] [answered]
The installation works, but affinity photo flickers a bit. How can I choose a different renderer? In the general documentations it provides a rum command when you can choose either gl or vulkan. How can I do the same on nixos?

**Comment by ghost** [comment]
Bump.
Usage: rum <wine-version or /path/to/wine/version> </path/to/wine/prefix> <wine command> [wine arguments...]
sudo rum /??? .local/share/affinity/ winetricks renderer =gl

what is the path to the wine version?

**Comment by mrshmllow** [comment, marked as answer]
```
nix run github:mrshmllow/affinity-nix#winetricks -- renderer =gl
```

Last section of the readme.

**Reply by ghost** [reply]
Is it really necessary to use nix run when using home-manager?

**Reply by mrshmllow** [reply]
I did not want to fill the path with 15 little packages, so left the adhoc stuff through nix run, but maybe thats something people want? You can just install the winetricks package if you want to.

**Reply by ghost** [reply]
Oh no, just thought I was maybe missing something and would have to look at the nix config. 

Thank you for making this awesome project! Would be cool if a lot of wine stuff would be managed through nix, instead of something like lutris.

**Reply by ghost** [reply]
> ```
> nix run github:mrshmllow/affinity-nix#winetricks -- renderer =gl
> ```
> 
> Last section of the readme.

And if anyone finds this, use 
nix run github:mrshmllow/affinity-nix#winetricks renderer =gl

instead of -- renderer =gl

**Reply by ghost** [reply]
Sor. i reinstalled and i now says
```
warning: You should update using your distribution's package manager, --self-update, or manually.

Using winetricks 20240105 - sha256sum: d852bb94aeb877fa47d2590c508a86f61eef30e88df67e7f70aa23dc48cdfeb1 with wine-9.14 and WINEARCH=win64
Unknown arg renderer

```

**Reply by l-konrad** [reply]
nvm just need to put = without space
